src/discljord/messaging/impl.clj

```diff
@@ -87,6 +100,18 @@
                [(keyword (str/replace (name k) #"-" "_")) v]))
         opts))
 
+(defmacro ^:private def-message-dispatch
+  "Creates a dispatch definition based on the common pattern of creating,
+  editing and updating some sort of message"
+  [name params method url]
+  (let [[opts status body] (repeatedly gensym)
+        delete (= method :delete)]
```

Comment:
Doesn't really matter here, and won't block me merging this, but a style thing is that this should have a question mark, maybe like `should-delete?`.

Reply:
Renamed to `delete?` rather than `should-delete?` because `delete` is the name of the subject method and not a verb.